app/questionnaire/placeholder_transforms.py

```diff
@@ -39,12 +39,13 @@ def format_date(self, date_to_format: str, date_format: str) -> str:
 
     @staticmethod
     def format_list(list_to_format: Sequence[str]) -> str:
-        formatted_list = "<ul>"
-        for item in list_to_format:
-            formatted_list += f"<li>{item}</li>"
-        formatted_list += "</ul>"
-
-        return formatted_list
+        if list_to_format is not None:
+            formatted_list = "<ul>"
+            for item in list_to_format:
+                formatted_list += f"<li>{item}</li>"
+            formatted_list += "</ul>"
+            return formatted_list
+        return ""
```

Comment:
I think this should be reverted, I can't see a real use case where we would ask to format an empty list and accept an empty string as being valid. Given there are no requirements around it and given it is not used, I don't think it should be changed. I think the list count you added is fine for the purposes of demoing the routing behaviour.